homeassistant/components/nmap_tracker/config_flow.py

```diff
@@ -26,25 +27,26 @@
 DEFAULT_NETWORK_PREFIX = 24
 
 
-def get_network():
+def get_network() -> str:
     """Search adapters for the network."""
-    adapters = ifaddr.get_adapters()
+    adapters: list[ifaddr.Adapter] = ifaddr.get_adapters()
     local_ip = get_local_ip()
     network_prefix = (
         get_ip_prefix_from_adapters(local_ip, adapters) or DEFAULT_NETWORK_PREFIX
     )
     return str(ip_network(f"{local_ip}/{network_prefix}", False))
 
 
-def get_ip_prefix_from_adapters(local_ip, adapters):
+def get_ip_prefix_from_adapters(local_ip: str, adapters) -> int | None:
```

Comment:
```suggestion
def get_ip_prefix_from_adapters(local_ip: str, adapters) -> int:
```

Reply:
I guess its possible that it could be `None`, but its likely unreachable unless someone installs a new network adapter between when the adapters are fetched get_local_ip is called